content/docs/09-registries-and-packs/3-add-custom-packs.md

```diff
@@ -35,7 +35,7 @@ Each pack contains a metadata file named `pack.json`. The table below explains i
 | layer | String | True | Relevant layer that this pack should be part of; such as os, k8s, cni, csi, addon |
 | addonType | String | False | Addon-type must be set for packs that have the layer set to Addon. The value must be one of the following: logging, monitoring, load balancer, authentication, ingress, security. Setting a relevant correct addon type ensures packs are organized correctly on the management console making it easy for profile authors to find packs. |
 | version | String | True | A Semantic version for the pack. It is recommended that the pack version be the same as the underlying integration it is being created for. For example, the version for the pack that will install Prometheus 2.3.4, should set to 2.3.4. |
-| cloudTypes | Array | True | Supported cloud types are AWS, Azure, VMware. One or more types can be provided for a pack. |
+| cloudTypes | Array | True | You can provide one or more types for a pack. Supported values are as follows: <br /><br />**all**, **aws**, **azure**, **gcp**, **tencent**, **vsphere**, **openstack**, **baremetal**, **maas**, **aks**, **eks**,  **tke**, **edge**, **edge-native**, **coxedge**, and **libvirt** (virtual edge and virtual clusters).
```

Comment:
"**libvirt** (virtual edge and virtual clusters)"

1. here libvirt is for virtualized edge
2.  we have a cloud type `edge` for a native edge.
3. for virtual clusters it's `all` as we deploy only add-ons in virtual clusters, infra layers are from the host cluster.

@brathina-spectro can you plz confirm the correctness of these observations?
cc: @karl-cardenas-coding

Reply:
@resmaa I checked with Boobalan, and he confirms that it should say 'libvirt (virtualized edge)'. I've made this change.